torchao/quantization/quant_api.py

```diff
@@ -1769,43 +1771,90 @@ class Float8DynamicActivationFloat8SemiSparseWeightConfig(AOBaseConfig):
     """
 
     layout: Layout = CutlassSemiSparseLayout()
-    activation_dtype: torch.dtype = e5m2_dtype
+    activation_dtype: torch.dtype = e4m3_dtype
     weight_dtype: torch.dtype = e4m3_dtype
+    granularity: Optional[Union[FP8Granularity, List[FP8Granularity]]] = PerRow()
+    activation_value_lb: Optional[float] = None
+    activation_value_ub: Optional[float] = None
+    float8_packing_format: Float8TensorPackingFormat = (
```

Comment:
version 2 with new packing_format should be added to `Float8DynamicActivationFloat8WeightConfig` I think?